core/router/downlink.go

```diff
@@ -49,12 +49,8 @@ func (r *router) UnsubscribeDownlink(gatewayID string) error {
 	return nil
 }
 
-func (r *router) HandleDownlink(downlink *pb_broker.DownlinkMessage) error {
+func (r *router) HandleDownlink(downlink *pb_broker.DownlinkMessage) (err error) {
```

Comment:
wanted to be consistent with all other functions of `handleXXX`  type(where the variable is used)
are you sure it has to be changed?